skglm/experimental/tests/test_sqrt_lasso.py

```diff
@@ -69,7 +70,10 @@ def test_PDCD_WS(with_dual_init):
 
     dual_init = y / norm(y) if with_dual_init else None
 
-    w = PDCD_WS(dual_init=dual_init).solve(X, y, SqrtQuadratic(), L1(alpha))[0]
+    datafit = compiled_clone(SqrtQuadratic())
+    penalty = compiled_clone(L1(alpha))
+
+    w = PDCD_WS(dual_init=dual_init).solve(X, y, datafit, penalty)[0]
```

Comment:
how was this working before if the datafit and penalties were not compiled ?

Reply:
They were compiled inside the solver.